src/js/lib/jquery.kazoosdk.js

```diff
@@ -492,7 +492,8 @@
 				'list': { verb: 'GET', url: 'accounts/{accountId}/users' },
 				'quickcall': { verb: 'GET', url: 'accounts/{accountId}/users/{userId}/quickcall/{number}' },
 				'hotdesks': { verb: 'GET', url: 'accounts/{accountId}/users/{userId}/hotdesks' },
-				'updatePresence': { verb: 'POST', url: 'accounts/{accountId}/users/{userId}/presence' }
+				'updatePresence': { verb: 'POST', url: 'accounts/{accountId}/users/{userId}/presence' },
+				'listDevicesByUser': { verb: 'GET', url: 'accounts/{accountId}/users/{userId}/devices' }
```

Comment:
Now that you've it under the `user` resource, the suffix `ByUser` does not make sense anymore.